kibana/Dockerfile

```diff
@@ -40,7 +39,22 @@ ADD probe/ /usr/share/kibana/probe/
 ADD lib/origin-kibana ${KIBANA_HOME}/installedPlugins/origin-kibana
 ADD kibana.yml ${KIBANA_CONF_DIR}/
 ADD run.sh utils install.sh prep-install.${RELEASE_STREAM} ${HOME}/
+ADD logo-OCP-console-hdr-thin.svg ${KIBANA_HOME}/installedPlugins/origin-kibana/public/images/logo-origin-thin.svg
 RUN sh ${HOME}/install.sh
 
 WORKDIR ${HOME}
 CMD ["./run.sh"]
+
+LABEL \
+        io.k8s.description="Kibana container for querying Elasticsearch for aggregated logs" \
+        com.redhat.component="logging-kibana-docker" \
+        vendor="Red Hat" \
+        name="openshift3/logging-kibana" \
+        License="GPLv2+" \
+        io.k8s.display-name="Kibana" \
+        version="v3.9.0" \
+        architecture="x86_64" \
+        release="0.46.0.0" \
```

Comment:
this shouldn't be "1" as well?

Reply:
All of this will be regenerated by cd tooling